README.md

```diff
@@ -84,7 +82,7 @@ can move to using the stable version.
 
 ## More Information
 
-* [Single Sign-On Guide](https://workos.com/docs/sso/guide)
-* [Directory Sync Guide](https://workos.com/docs/directory-sync/guide)
-* [Admin Portal Guide](https://workos.com/docs/admin-portal/guide)
-* [Magic Link Guide](https://workos.com/docs/magic-link/guide)
+- [Single Sign-On Guide](https://workos.com/docs/sso/guide)
+- [Directory Sync Guide](https://workos.com/docs/directory-sync/guide)
+- [Admin Portal Guide](https://workos.com/docs/admin-portal/guide)
+- [Magic Link Guide](https://workos.com/docs/magic-link/guide)
```

Comment:
I must have an on-save Markdown formatter installed, though I don't remember installing one. Maybe it was one of our recommended plugins?

In any case this is probably a good change. We don't upload this README to [nuget](https://www.nuget.org/packages/WorkOS.net) so no worries about formatting inconsistencies.